Jobs [Distributed Scheduler] API docs, pt.1 (#4164)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/scheduler/howto-use-scheduler.md

---
type: docs
title: "How-To: Set up a scheduler"
linkTitle: "How-To: Set up a scheduler"
weight: 2000
description: "Learn how to use the scheduler API to schedule jobs"
---

> Now that you've learned what the [Dapr scheduler building block]({{< ref scheduler-overview.md >}}) provides, learn how it can work in your environment. The below code example loosely describes an application that schedules and orchestrates an application.

Reviewer: ```suggestion
Now that you've learned what the [scheduler building block]({{< ref scheduler-overview.md >}}) provides, let's look at an example of how to use the API. The below code example describes an application that schedules and orchestrates XXXX.
```
